Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -29,7 +29,7 @@ namespace VertexLoaderManager
 {
 
 float position_cache[3][4];
-u32 position_matrix_index[3];
+u32 position_matrix_index[4]; // only 1-3 are used. 0 is for SSE access by the vertex loader JITs
```

Comment:
It's not actually SSE, it's just that the counter added to the address of the array is 1, 2, or 3, but never zero (that's why the old code computed an out-of-bounds pointer one byte before the array).